Src/FluentAssertions/Equivalency/SelfReferenceEquivalencyAssertionOptions.cs

```diff
@@ -312,6 +316,26 @@ public TSelf ExcludingProperties()
             return (TSelf)this;
         }
 
+        /// <summary>
+        /// Instructs the comparison to include non-browsable members (members with an EditorBrowsableState of Never).
+        /// </summary>
+        /// <returns></returns>
+        public TSelf IncludingNonBrowsableMembers()
```

Comment:
I was looking at the existing `IncludedFields` and `IncludedProperties` members -- I don't see how those can be applied to just certain types. What am I missing??

Reply:
Like `ComparingByMembers<T>` and `ComparingByMembers(Type type)`

Reply:
Hmm, that feels like it might be overcomplicating things a bit. Is it likely that someone will have an object graph where they need to compare non-browsable properties in one part of the graph and not in another part of the same graph??

Reply:
> Is it likely that someone will have an object graph where they need to compare non-browsable properties in one part of the graph and not in another part of the same graph??

That's exactly what I was wondering. I haven't used that attribute myself a lot, so I can't judge that. Do you?